cypress/integration/ui_spec.js

```diff
@@ -1172,4 +1172,53 @@ describe("UI", () => {
 			});
 		});
 	});
+	describe("localstorage", () => {
+		it("should only have device and account registration", () => {
+			const parleyConfig = {userAdditionalInformation: {"some-key": "some-value"}};
+			const testMessage = `test message localstorage variable check ${Date.now()}`;
+			visitHome(parleyConfig);
+
+			cy.get("[id=app]").as("app");
+
+			clickOnLauncher();
+			sendMessage(testMessage);
+
+			const newAuthHeader = "1234";
+			const apiVersion = "v1.6";
+
+			// see configuration options https://developers.parley.nu/docs/settings#runoptions
+			cy.window().then((win) => {
+				// eslint-disable-next-line no-param-reassign
+				win.parleySettings.authHeader = newAuthHeader;
+				// eslint-disable-next-line no-param-reassign
+				win.parleySettings.apiVersion = apiVersion;
+			});
+			cy.window().then((win) => {
+				// get the content of the deviceInformation key from the local storage
+				let localStorageValue = win.localStorage.getItem("deviceInformation");
+				localStorageValue = JSON.parse(localStorageValue);
+				// eslint-disable-next-line no-unused-expressions
+				expect(localStorageValue.accountIdentification).to.exist;
+				// eslint-disable-next-line no-unused-expressions
+				expect(localStorageValue.deviceIdentification).to.exist;
+
+				// eslint-disable-next-line no-unused-expressions
```

Comment:
Waarom is dit nodig?
Ik zie andere plekken waar `expect()` gebruikt wordt geen eslint disable
bv https://github.com/parley-messaging/web-library/blob/63afa4e844ca10d891c8709443e1b219e2516438/cypress/integration/ui_spec.js#L561

Reply:
Vermoed dat ESLint bij het verifiëren de suggesties door heeft gevoerd

Reply:
chai assertions binnen cypress zoals expect.to.be.true etc werken niet fijn met ESlint. Ik heb daarom de .expect veranderd naar .should. Zie: c496b6c2438881c4762da3d20e5b4a24f6b01b96